doc: Add mokosmart gateway (#327)
TheThingsIndustries/lorawan-stack-docs: doc/content/gateways/moko-mkgw2-lw/_index.md

---

> title: "MOKOSMART MKGW2-LW LoRaWAN Gateway"

Reviewer: Their webpage says MOKOSmart

Author: @Fakerallen wrote it so I'm going to leave it

description: ""
---

MOKOSMART's 8 channel MKGW2 gateway is embedded with Semtech’s high performance multi-channel transceiver SX1301/SX1257 and MTK platform. It is for indoor use and is easy to install.

> {{< figure src="mkgw2-lw.jpg" alt="mkgw2-lw" class="plain float" >}}

Reviewer: This image is saved with a space before .jpg, just rename it

## Requirements

1. User account on {{% tts %}} with rights to create gateways.

> 2. MOKOSMART MKGW2-LW connected to the internet via Ethernet or WiFi. (For Ethernet and WiFi settings, see sections 5 and 6 of the [user manual](https://www.mokosmart.com/lorawan-gateway-mkgw2-lw/))

Reviewer: I don't see a user manual here, this is a product overview page

Author: @Fakerallen

## Registration

> Create a gateway by following [the instructions]({{< ref "/gateways/adding-gateways" >}}). 

Reviewer: ```suggestion
Create a gateway by following the instructions [Adding Gateways]({{< ref "/gateways/adding-gateways" >}}). 
```

> The **EUI** of the gateway can be found on the configuration web page of the gateway. 

Reviewer: ```suggestion
The **EUI** of the gateway can be found under **Server Access &#8594; Gateway ID** on the gateway web configuration platform. 
```

{{< figure src="eui.png" alt="Gateway EUI" >}}

> ## Configuration via Browser

Reviewer: This is not really a configuration, but accessing the gateway via web platform only. I think this section should come before Registration, because finding the EUI in the web management platform requires to be able to access it first

Author: We always add the gateway in the console first. For Basic Station, you need to generate an API key for the device to configure it. The EUI is also printed on the packaging, @Fakerallen just didn't mention it.

1. Turn on the gateway and wait for about 60s.

> 2. Using your PC or phone, connect to the WiFi Access Point exposed by the gateway. The default SSID format is "MKGW2-LW-xxxx”, e.g "MKGW2-LW-91D8 ", where “91D8” is the last two bytes of the gateway MAC address. The default password is Moko4321.

Reviewer: ```suggestion
2. Using your PC or phone, connect to the WiFi Access Point exposed by the gateway. The default SSID format is `MKGW2-LW-xxxx`, e.g `MKGW2-LW-91D8`, where `91D8` are the last two bytes of the gateway MAC address. The default password is `Moko4321`.
```

> 3. After successful connection, the WEB management platform can be accessed through the IP address 192.168.22.1 of the gateway LAN interface.

Reviewer: ```suggestion
3. After a successful connection, the web management platform can be accessed through the IP address `192.168.22.1` of the gateway LAN interface.
```

> 4. Log in using the following default credentials:
>   Username: Admin 
>   Password: admin

Reviewer: ```suggestion
4. Log in using the default credentials - username `Admin` and password `admin`
```

> It is recommended that the default password is changed for security reasons.

Reviewer: ```suggestion

It is recommended that the default password is changed for security reasons.
```

> {{< figure src="login in web.png" alt="Login In Web" >}}

Reviewer: Rename the image, this doesn't work

## Connect to {{% tts %}}

The MOKOSMART MKGW2-LW supports the UDP packet forwarder. Please follow instructions for connecting the [UDP packet forwarder]({{< ref "gateways/semtech-udp-packet-forwarder" >}}).

{{< figure src="eui.png" alt="Gateway EUI" >}}

> Don't forget to click "SAVE&APPLY" after you fill in the parameters. After a few seconds, your gateway should connect to {{% tts %}}.

Reviewer: ```suggestion
Do not forget to click **SAVE&APPLY** after you fill in the parameters. After a few seconds, your gateway should connect to {{% tts %}}.
```

> For more information, you can refer to the MKGW2-LW user manual or send an email to [support_lora@mokotechnology.com](mailto:support_lora@mokotechnology.com).

Reviewer: ```suggestion
For more information, you can refer to the MKGW2-LW user manual or email the [MOKO Technology support](mailto:support_lora@mokotechnology.com).
```
